unittests/resources/checks/frontend_checks.py

```diff
@@ -80,6 +80,22 @@ def __init__(self):
         }
 
 
+@rfm.simple_test
+class PerformanceSuccessCheck(BaseFrontendCheck):
```

Comment:
The performance failure check should be enough. You don't need this. The performance report will be produced even in case of failures.